src/main/java/com/mitchej123/hodgepodge/LoadingConfig.java

```diff
@@ -280,6 +281,7 @@ public LoadingConfig(File file) {
         validatePacketEncodingBeforeSendingShouldCrash = config.get(Category.FIXES.toString(), "validatePacketEncodingBeforeSendingShouldCrash", false, "Should the extended packet validation error cause a crash (true) or just print out an error to the log (false)").getBoolean();
 
         compactChat = config.get(Category.TWEAKS.toString(), "Compact chat", true, "Compacts identical consecutive chat messages together").getBoolean();
+        dontInvertCrosshairColor = config.get(Category.TWEAKS.toString(), "dontInvertCrosshairColor", true, "Stop inverting colors of crosshair").getBoolean();
```

Comment:
Please disable it by default as this make the crosshair hard to see on very bright things.